_api_app/app/Http/routes.php

```diff
@@ -55,6 +58,8 @@
                 $app->put('entries', 'SectionEntriesController@order');
                 $app->delete('entries', 'SectionEntriesController@delete');
                 $app->put('entries/galleries', ['as' => 'entry_gallery', 'uses' => 'SectionEntriesController@galleryOrder']);
+                $app->post('entries/galleries', 'SectionEntriesController@galleryUpload');
```

Comment:
The naming of routes has gone out of whack - it's unrepresentative for the methods in controllers. We should fix it ASAP, but I guess it's out of the scope of this PR.